README.md

```diff
@@ -71,15 +83,23 @@ console.log(doc.info().title());
 // => Example
 ```
 
-##### Example using OpenAPI schemas
+### Example using OpenAPI schemas
 
 Head over to [asyncapi/openapi-schema-parser](https://www.github.com/asyncapi/openapi-schema-parser) for more information.
 
-##### Example using RAML data types
+### Example using RAML data types
 
 Head over to [asyncapi/raml-dt-schema-parser](https://www.github.com/asyncapi/raml-dt-schema-parser) for more information.
 
-### Custom message parsers
+### Example using Avro schemas
```

Comment:
I think we should move Avro to become the first one. In messaging world, it's much more extended than OpenAPI and RAML.